scripts/download_elevation_grid.py

```
@@ -0,0 +1,484 @@
+    ) -> List[float]:
+        """Download elevations for a batch of points."""
+        elevations = []
+        
+        if use_api and len(points) <= 100:
+            timeout = aiohttp.ClientTimeout(total=10)
+            async with aiohttp.ClientSession(timeout=timeout) as session:
+                try:
+                    locations = "|".join([f"{lat},{lon}" for lat, lon in points])
+                    async with session.get(
+                        self.open_elevation_api,
+                        params={'locations': locations}
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)